systems/plants/TimeSteppingRigidBodyManipulator.m

```diff
@@ -118,6 +119,7 @@ function checkDirty(obj)
 
     function model = compile(model)
       w = warning('off','Drake:RigidBodyManipulator:UnsupportedContactPoints');
+      model.gurobi_present = checkDependency('gurobi');
```

Comment:
This call to gurobi is how we derived the minimal crash discussed here: https://github.com/mitdrc/drc/pull/1251#issuecomment-89026122